[2024-06-19T08:35:42.692+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T08:35:42.770+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-06-19T08:35:31.134261+00:00 [queued]>
[2024-06-19T08:35:42.789+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-06-19T08:35:31.134261+00:00 [queued]>
[2024-06-19T08:35:42.790+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-19T08:35:42.842+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_wikipedia_data> on 2024-06-19 08:35:31.134261+00:00
[2024-06-19T08:35:42.857+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'wikipedia_flow', 'transform_wikipedia_data', 'manual__2024-06-19T08:35:31.134261+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/wikipedia_flow.py', '--cfg-path', '/tmp/tmpas7q7iq2']
[2024-06-19T08:35:42.862+0000] {standard_task_runner.py:91} INFO - Job 25: Subtask transform_wikipedia_data
[2024-06-19T08:35:42.870+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=3370) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-19T08:35:42.872+0000] {standard_task_runner.py:63} INFO - Started process 3372 to run task
[2024-06-19T08:35:42.978+0000] {task_command.py:426} INFO - Running <TaskInstance: wikipedia_flow.transform_wikipedia_data manual__2024-06-19T08:35:31.134261+00:00 [running]> on host dd278de8866b
[2024-06-19T08:35:43.243+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ayush Goswami' AIRFLOW_CTX_DAG_ID='wikipedia_flow' AIRFLOW_CTX_TASK_ID='transform_wikipedia_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T08:35:31.134261+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-19T08:35:31.134261+00:00'
[2024-06-19T08:35:43.245+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T08:35:51.876+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=Wembley+Stadium+%0A%2C%C2%A0England%0A&format=json&limit=1
[2024-06-19T08:36:39.149+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=C%C3%ADvitas+Metropolitano%2C%C2%A0Spain&format=json&limit=1
[2024-06-19T08:37:18.760+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=Stadion+Narodowy+%2C%C2%A0Poland&format=json&limit=1
[2024-06-19T08:38:30.614+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=Estadio+Jos%C3%A9+Amalfitani%2C%C2%A0Argentina%0A&format=json&limit=1
[2024-06-19T08:39:10.839+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=Suwon+World+Cup+Stadium%2C%C2%A0South+Korea%0A&format=json&limit=1
[2024-06-19T08:39:48.975+0000] {connectionpool.py:874} WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Read timed out. (read timeout=1)")': /search?q=Stadion+Lukas+Enembe%2C%C2%A0Indonesia&format=json&limit=1
[2024-06-19T08:40:33.267+0000] {logging_mixin.py:188} WARNING - /opt/***/dags/pipeline/wikipedia_pipeline.py:76 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-06-19T08:40:33.405+0000] {python.py:237} INFO - Done. Returned value was: ok
[2024-06-19T08:40:33.406+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T08:40:33.558+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=wikipedia_flow, task_id=transform_wikipedia_data, run_id=manual__2024-06-19T08:35:31.134261+00:00, execution_date=20240619T083531, start_date=20240619T083542, end_date=20240619T084033
[2024-06-19T08:40:33.707+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T08:40:33.740+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T08:40:33.768+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
